### ISSUE TITLE
Is there any way to save configuration locally on Rpanion?

### ISSUE THREAD

**Issue by lida2003** [stephendade/Rpanion-server#197]
I think it would be very nice if Rpanion can save the configuration, which was lastly executed sucessfully .

e.g. rtsp streamer

Currently it will fallback to 1920x1080 10Hz rate, 1000kbps, default not started.

Is it possible for Rpanion to remember what I set  last time. 1280 x 720 50Mbps 30FPS default started.

**Comment by stephendade** [comment]
Thanks for checking. This feature _should_ work, but I've checked and it doesn't work anymore.

I'll investigate and fix.

**Comment by stephendade** [comment]
#199 should fix the issue of the configuration not saving if streaming if active.

If streaming is not active, the configuration will not be saved.

**Comment by lida2003** [comment]
> #199 should fix the issue of the configuration not saving if streaming if active.
> 
> If streaming is not active, the configuration will not be saved.

I don't think so, when I reboot from ssh, it didn't work the way as it supposed to do.

```
$ git log -n 1
commit f8766efd1ef0ff4af6fc9c1e6536e968ec73e4da (HEAD -> master, origin/master,                                                                                                                  origin/HEAD)
Author: Stephen Dade <stephen_dade@hotmail.com>
Date:   Sun Jan 21 14:43:44 2024 +1100

    Video: Latency improvements

```

![图片](https://github.com/stephendade/Rpanion-server/assets/1296171/ae7b593c-2d01-409c-8da2-9c941400cded)

**Comment by lida2003** [comment]
Updated latest version, not good! Just check the below video.

 ```
$ git log -n 1
commit d066b49d67d32c3307be00f006e0ecaefcf93b19 (HEAD -> master, origin/master, origin/HEAD)
Author: Stephen Dade <stephen_dade@hotmail.com>
Date:   Mon Feb 5 16:09:05 2024 +1100

    Video: add correct values for testvideosrc and jetson CSI

```


https://github.com/stephendade/Rpanion-server/assets/1296171/8d06fd5c-7de5-49de-8c66-77b4823df300

**Comment by stephendade** [comment]
Yep, I think my latest commits broke a few things ... I will investigate and get back to you.

**Comment by stephendade** [comment]
It should be fixed in #202 (now merged into master). See if that works for you.

**Comment by lida2003** [comment]
> It should be fixed in #202 (now merged into master). See if that works for you.

I works for me, Thanks.

![图片](https://github.com/stephendade/Rpanion-server/assets/1296171/f3aca7c6-8417-4120-b132-156fb78e666d)
